src/main/java/edu/nextstep/racingcar/step2/CalculateCode.java

```diff
@@ -0,0 +1,21 @@
+package edu.nextstep.racingcar.step2;
+
+public enum CalculateCode {
+
+    ADD("+"), SUB("-"), MULTI("*"), DIV("/");
```

Comment:
```suggestion
    ADD("+", Math::addExact),
    SUB("-", Math::subtractExact),
    MULTI("*", Math::multiplyExact),
    DIV("/", (a, b) -> a / b);
```
이런 형태로 표현하고 관리해주면 어떨까요?

Reply:
enum 에 로직이 포함되면, 코드의 가독성이 떨어지지 않을까요?